src/main/webapp/content/env.js

```diff
@@ -1,2 +1,2 @@
 var baseUrl = window.location.href.split('/content')[0];
-window.serverUrl = baseUrl;
+window.serverUrl = baseUrl;
```

Comment:
do you still need this file, since there are `ui/env.development.js` and `ui/env.production.js`

Reply:
it is still needed because that is the config for the ContentUI